source/documentation/index.html.haml

```diff
@@ -58,51 +58,22 @@ wiki_last_updated: 2014-04-26
 
   .col-md-4
     :markdown
-      ### Developer Documentation
-
-      - [Install nightly snapshot](/develop/dev-process/install-nightly-snapshot)
-      - [Building oVirt engine](/develop/developer-guide/engine/engine-development-environment)
-      - [Building oVirt Node](/develop/projects/node/building)
-      - [Building VDSM](/develop/developer-guide/vdsm/developers)
-      - [Contributing to the Node project](/develop/projects/node/contributing-to-the-node-project)
-      - [Submitting a patch with Gerrit](/develop/dev-process/working-with-gerrit)
-      - [The development process](/develop/dev-process/devprocess)
-      - [Release management](/develop/release-management/releases)
-      - [Getting in contact with the oVirt community](/community/about/contact)
-      - [Becoming a maintainer](/develop/dev-process/becoming-a-maintainer)
-      - [oVirt architecture](/documentation/architecture/architecture)
-      - [Building a custom user portal](/develop/developer-guide/sample-user-portals)
-      - [Building oVirt engine DWH](how-to/reports/dwh-development-environment)
-      - [Building oVirt engine Reports](how-to/reports/reports)
+      ### Additional Documentation
 
+      - [Introduction](./introduction)
+      - [Administration](./admin-guide)
+      - [Architecture](./architecture)
+      - [How To](./how-to)
+      - [Internal](./internal)
+      - [Security](./security)
+      - [SLA](./sla)
+      - [Storage](./storage)
+      - [Drafts](./draft)
+      - [ּresources](./resources)
```

Comment:
You edited my comment by mistake :-/

Anyway, there is a character just before the small 'r' of "resources". It looks to me as a very small dot at mid-vertical position.

It seems to be this one:
U+05BC ( ◌ּ ) {HEBREW POINT DAGESH OR MAPIQ}